ubuntu-frame/README.md

```diff
@@ -0,0 +1,216 @@
+# TensorFlow Lite Examples snap for Ubuntu Core
+
+This snap bundles example TensorFlow Lite applications with all their required dependencies to run on Ubuntu Core.
+They are based on the
+[TensorFlow Lite examples](https://github.com/tensorflow/examples/tree/master/lite/examples).
+
+Core24, which is used for the base of this snap, ships with Python 3.12.
+TensorFlow Lite does not currently work with Python 3.12 due to dependencies that are [not yet updated](https://github.com/tensorflow/tensorflow/issues/62003).
+The most reliable combination of dependencies that work on AMD64 and ARM64 are based on Python 3.8.
+This snap includes Python 3.8 from the [deadsnakes PPA](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa).
+
+This snap was tested on the following platforms, but should work on any platform that runs SnapD:
+
+- Ubuntu Desktop 24.04 on AMD64 workstation
+- Ubuntu Server 24.04 on Raspberry Pi 5
+- Ubuntu Core 24 on Raspberry Pi 5
+
+## Build the snap
+
+This snap is not available via the Snap Store.
+You will need to build it from source yourself.
+Clone this repository and then run the following command in the cloned directory:
+
+```
+snapcraft -v
+```
+
+We recommend building on Ubuntu Desktop 24.04 or Ubuntu Server 24.04.
+
+## Install the snap
+
+```
+snap install --dangerous ./tf-custom-examples_*.snap
+```
+
+You also need to connect the camera plug for the camera examples to work:
+
+```
+sudo snap connect tf-custom-examples:camera
+```
+
+The snap includes a daemon process called `camera-detect-stream` that automatically starts when the snap is installed.
+Stop it now until you read the rest of the readme.
+
+```
+sudo snap stop tf-custom-examples.camera-detect-stream
+```
```

Comment:
Could set `apps.<app-name>.install-mode` to `disable` so it doesn't start by default.